internal/resources/appplatform/resource.go

```diff
@@ -510,6 +520,8 @@ func SaveResourceToModel[T sdkresource.Object](
 		}
 	}
 
+	dst.ID = meta.UID
```

Comment:
Since `meta.uid` is not unique across time (e.g. an object X with `uid = "foo"` can be created and then deleted, after which object Y can be again created with the same `uid), I think it's best to use `UUID`, which is unique across space & time:
```suggestion
	dst.ID = meta.UUID
```

Reply:
Updated!